src/saveload/saveload.cpp

```diff
@@ -1043,45 +1043,28 @@ static void SlStdString(void *ptr, VarType conv)
 }
 
 /**
- * Return the size in bytes of a certain type of atomic array
- * @param length The length of the array counted in elements
- * @param conv VarType type of the variable that is used in calculating the size
+ * Internal function to save/Load a list of SL_VARs.
+ * SlCopy() and SlArray() are very similar, with the exception of the header.
```

Comment:
Just for confirmation:
SlCopy is "meant" for anything that has a dynamic size?
SlArray is "meant" for anything that has a fixed size?
Or am I missing something else here? I can't really figure out from the comments of either SlCopy or SlArray.
And I'm seeing SlCopies with fixed sizes, so I guess what I'm thinking is wrong. In any case, I guess it needs to be clearer whether when to use SlCopy or SlArray.

Reply:
Will update the comments, but basically:

`SlCopy` you can use from outside `saveload.c`, `SlArray` is only usable via an `SL_ARR`. The only reason for the difference is that one has a length-field and the other doesn't.